nestkernel/nest_names.cpp

```diff
@@ -366,5 +366,15 @@ const Name xs( "xs" );
 
 const Name z( "z" );
 const Name z_connected( "z_connected" );
+
+//time-reversal state-space-spike detection algorithm for precise
+//neuron models
+
+const Name y1_ex("y1_ex");
+const Name y1_in("y1_in");
+const Name y0("y0");
+const Name y2("y2");
```

Comment:
I don't think any of these new names are necessary; please use existing names, see comments below. Names should be inserted in strictly alphabetical order.